src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/Schemas/JSON/template.json

```diff
@@ -1027,7 +1027,7 @@
             "properties": {
               "type": {
                 "description": "The symbol binds value from external sources.",
-                "enum": [ "bind" ]
+                "enum": ["bind"]
```

Comment:
is it `editorconfig` supported enforcement?  
can we add a rule for this so editor behaves correctly?

Reply:
Asked here as well: https://github.com/SchemaStore/schemastore/pull/2255#issuecomment-1307430739